src/__tests__/ParseUser-test.js

```diff
@@ -675,6 +672,71 @@ describe('ParseUser', () => {
     });
   });
 
+  it('can retreive a user with sessionToken (me)', async () => {
+    ParseUser.disableUnsafeCurrentUser();
+    ParseUser._clearCache();
+    CoreManager.setRESTController({
+      request(method, path, body, options) {
+        expect(method).toBe('GET');
+        expect(path).toBe('users/me');
+        expect(options.sessionToken).toBe('123abc');
+        return Promise.resolve({
+          objectId: 'uid3',
+          username: 'username',
+          sessionToken: '123abc'
+        }, 200);
+      },
+      ajax() {}
+    });
+
+    const user = await ParseUser.me('123abc');
+    expect(user.id).toBe('uid3');
+    expect(user.isCurrent()).toBe(false);
+    expect(user.existed()).toBe(true);
+  });
+
+  it('can retreive a user with sessionToken and masterKey(me)', async () => {
```

Comment:
Is there a use case for this?  In other words, how would this request differ from the above?  The master key is included, but does it behave it any differently?  I looked at the router code, but not sure...